target/linux/ramips/dts/mt7628an_wavlink_wl-wn577a2.dts

```diff
@@ -0,0 +1,117 @@
+/dts-v1/;
```

Comment:
Please add an SPDX license identifier before this and an empty line after.

Reply:
Add empty link. But I'm confused, other dts don't have an SPDX license identifier...

Reply:
Again, we have started to require this add a certain point. And it's not so easy to add (=change) a license on an existing file.

Reply:
Ok, so adding /MIT/ right after /dts-v1/ would fulfill the new requirement? I just grep'ed through the dts files and didn't find one with and license identifier...

Reply:
Or should it state /SPDX-License-Identifier: GPL-2.0-only/?

Reply:
You will end up with something like this:
https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7628an_tplink_tl-wr841n-v14.dts

I would call that the preferred license as well. But you may choose your own license identifier (as far as it's reasonable).